README.md

```diff
@@ -1,12 +1,27 @@
-# Tsickle - TypeScript to Closure Annotator [![Build Status](https://travis-ci.org/angular/tsickle.svg?branch=master)](https://travis-ci.org/angular/tsickle)
+# Tsickle - TypeScript to Closure Translator [![Build Status](https://travis-ci.org/angular/tsickle.svg?branch=master)](https://travis-ci.org/angular/tsickle)
 
-Tsickle processes TypeScript and adds [Closure Compiler]-compatible JSDoc
-annotations. This allows using TypeScript to transpile your sources, and then
+Tsickle converts TypeScript code into a form acceptible to the [Closure Compiler].
+This allows using TypeScript to transpile your sources, and then using
 Closure Compiler to bundle and optimize them, while taking advantage of type
 information in Closure Compiler.
 
 [Closure Compiler]: https://github.com/google/closure-compiler/
 
+## What conversion means
+
+A (non-exhaustive) list of the sorts of transformations Tsickle applies:
+
+- inserts Closure-compatible JSDoc annotations on functions/classes/etc
+- converts ES6 modules into `goog.module`/`goog.provide`
```

Comment:
Part of the point of tsickle is for users to need to know about goog.module modules. :)  (In principle if Closure Compiler improved its ES6 support we may not even do this transformation.)

The point of this wall of text is instead just to say "tsickle does some invasive code transformations, it doesn't just insert jsdoc" (which is what outsiders often think when they're not familiar with the guts).